### PR TITLE
Unreal docs staging

### PR DESCRIPTION
**Pull request by ereynoldsML**
Updating Unreal preview docs.

### CODE REVIEW

File: docs/guides/unreal/unreal-getting-started.md [hunk 1]
```diff
@@ -2,70 +2,68 @@
...
+You’ll be building the Unreal Engine from source code, which can take several hours or more.
 :::
 
 ## Setup Unreal Engine and the Magic Leap 2 Unreal SDK
 
 The version of Unreal Engine for this preview is located in a fork of the Epic Games Unreal Engine GitHub repository. The Magic Leap 2 Unreal SDK is available from the Magic Leap Hub.
 
+### Get access to the Unreal Engine repository
+
+To get access to the Epic Games Unreal Engine GitHub repository, follow the instructions on this page on the Unreal Engine website: [[Accessing Unreal Engine source code on GitHub]](https://www.unrealengine.com/en-US/ue-on-github).
```
Comment (Babilinski) [inline docs/guides/unreal/unreal-getting-started.md:34]: replace

```
[[Accessing Unreal Engine source code on GitHub]](https://www.unrealengine.com/en-US/ue-on-github).
```

with 

```
[Accessing Unreal Engine source code on GitHub](https://www.unrealengine.com/en-US/ue-on-github).
```

File: docs/guides/unreal/unreal-getting-started.md [hunk 2]
```diff
@@ -2,70 +2,68 @@
...
 
 You’ll need the Magic Leap Hub to download the Magic Leap 2 Unreal SDK and examples. If you don’t already have the Magic Leap Hub installed:
 
-1. Go to the [developer tool download page](https://ml2-developer.magicleap.com/downloads) on the Magic Leap 2 developer portal and install the Windows version of the Magic Leap Hub.
+1. Go to the [developer tools download page](https://ml2-developer.magicleap.com/downloads) on the Magic Leap 2 developer portal and install the Windows version of the Magic Leap Hub.
 2. Run the installer and complete the setup wizard.
-3. Install the redistributables from the [Microsoft Visual C++ Redistributable latest supported downloads](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist) page.
-
-### Install the ML C SDK (optional)
+3. Go to the [Microsoft Visual C++ Redistributable latest supported downloads](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist) page. Install the X64 redistributables for Visual Studio 2015, 2017, 2019, and 2022.
```
Comment (Babilinski) [inline docs/guides/unreal/unreal-getting-started.md:42]: Does it support all of the following versions or does it require that all 4 versions are installed?

File: docs/guides/unreal/unreal-getting-started.md [hunk 3]
```diff
@@ -141,47 +127,46 @@ mklink /d MagicLeap $UNREAL-SDK\Plugins
...
-The Magic Leap Application Simulator is not supported for this preview.  You can use Magic Leap Remote Rendering to run the apps you create on a desktop computer and stream it to the Magic Leap 2 device.
-
-You can find instructions for setting up Magic Leap Remote Rendering and using it with Unreal Engine 5 on the Magic Leap 2 developer portal. See [Remote Rendering service](/docs/guides/remote-rendering/remote-rendering) and [Remote Rendering with Unreal Engine 5](/docs/guides/remote-rendering/remote-render-unreal-engine-5).
-
-## Integration testing
-
-Find the integration test in the UnrealSDK project under `Projects/IntegrationTests` in the `unreal-sdk` folder. This is a host application for a suite of integration tests. Depending on the type of API you are implementing, you should either add a new map to this test, or create a new project, or both.
-
-## View logcat
+The executable file created is located in the `UnrealEngine` folder at \Engine\Binaries\Win64\UE5Editor.exe.
```
Comment (Babilinski) [inline docs/guides/unreal/unreal-getting-started.md:153]: Update the file path to monospace:

The executable file created is located in the `/UnrealEngine/` root directory under `/Engine/Binaries/Win64/UE5Editor.exe`.

File: docs/guides/unreal/unreal-getting-started.md (hunk 2 above)
Comment (ereynoldsML) [reply docs/guides/unreal/unreal-getting-started.md:42]: @Babilinski The Microsoft site as one download package for Visual Studio 2015, 2017, 2019, and 2022.

File: docs/guides/unreal/unreal-getting-started.md (hunk 3 above)
Comment (ereynoldsML) [reply docs/guides/unreal/unreal-getting-started.md:153]: Done.

File: docs/guides/unreal/unreal-getting-started.md (hunk 1 above)
Comment (ereynoldsML) [reply docs/guides/unreal/unreal-getting-started.md:34]: Done.

Comment (Babilinski) [reply docs/guides/unreal/unreal-getting-started.md:34]: When you make the changes do you mind running `git push` to upload the latest changes

Comment (ereynoldsML) [reply docs/guides/unreal/unreal-getting-started.md]: You should see them now.

File: docs/guides/unreal/unreal-getting-started.md [hunk 4]
```diff
@@ -2,74 +2,74 @@
...
 sidebar_position: 1
-date: 08/30/2023
+date: 08/31/2023
 tags: [Android,Unreal Engine,Getting Started]
 keywords: [Android,Unreal Engine,Getting Started]
 
 ---
 
-This preview version of the Magic Leap 2 SDK for Unreal Engine 5 (Magic Leap 2 Unreal SDK) lets you create apps for Magic Leap 2 devices using Unreal Engine, the 3D graphics engine created by Epic Games.
+This preview version of the Magic Leap 2 SDK for Unreal Engine 5 (Magic Leap 2 Unreal SDK) lets you create apps for Magic Leap 2 devices using Unreal Engine, the game engine created by Epic Games.
```
Comment (aheaney) [inline docs/guides/unreal/unreal-getting-started.md:11]: nit- one or the other

File: docs/guides/unreal/unreal-getting-started.md [hunk 5]
```diff
@@ -2,74 +2,74 @@
...
 
-Most functionality in the Magic Leap 2 Unreal SDK is exposed through OpenXR. Some optional APIs, like extended controller and camera functionality, require the Magic Leap Native API SDK (Magic Leap 2 C SDK), also called the C API. Some plugins expose their functionality through components and Blueprint function libraries.
+Most functionality in the Magic Leap 2 Unreal SDK is exposed through OpenXR rather than requiring the Magic Leap Native API SDK (Magic Leap 2 C SDK), also called the C API. Some optional APIs, like extended controller and camera functionality, require the Magic Leap 2 C SDK. Some plugins expose their functionality through components and Blueprint function libraries.
```
Comment (aheaney) [inline docs/guides/unreal/unreal-getting-started.md:13]: "Magic Leap Native API SDK (Magic Leap 2 C SDK), also called the C API"
can just call it one thing- maybe MagicLeap native C api

If we're going to try to explain this, we could clarify that the reason for this is some ML platform functionality is not yet exposed through OpenXR apis. Thus, some plugins must depend on ML c api.

Comment (aheaney) [inline docs/guides/unreal/unreal-getting-started.md:13]: On this note- out of scope for the guide here, but in theory we should provide documentation on what each of the plugins is for and how to use it. Perhaps we could at least have a table somewhere with the list of plugins and what each of them is for.

File: docs/guides/unreal/unreal-getting-started.md [hunk 6]
```diff
@@ -2,74 +2,74 @@
...
+- This preview release is supported only on Windows. For information on system requirements for running Unreal Engine 5 on Windows, see [Hardware and Software Requirements](https://docs.unrealengine.com/5.1/en-US/onboarding-guide-for-unreal-engine-games-licensees/) in the Unreal Engine 5 documentation site.
+- To clone the Magic Leap fork of the Unreal Engine source code for this preview, you need a public GitHub account and an Epic Games account.
+- Visual Studio 2022 is recommended. Visual Studio 2019 is also supported. If you want to use Visual Studio 2019, customize your Unreal Engine GenerateProjectFiles.bat file for that version. If you haven’t installed Visual Studio for use with Unreal Engine before, see  [Setting Up Visual Studio](https://docs.unrealengine.com/5.1/en-US/setting-up-visual-studio-development-environment-for-cplusplus-projects-in-unreal-engine/) in the Unreal Engine 5 documentation site.
```
Comment (aheaney) [inline docs/guides/unreal/unreal-getting-started.md:25]: I'd consider making this a little more affirmative. Like we should either tell people to go through Epic's guide or summarize the VS requirements here as there's a specific set of workloads and packages that you need to install.

Comment (aheaney) [inline docs/guides/unreal/unreal-getting-started.md:25]: "customize your UE GenerateProjectFiles.bat file"
If we're going to suggest this, we should probably say specifically what changes to make to the file. At this point in reading the guide, users also wouldn't have encountered the UE project source and wouldn't have the file.

File: docs/guides/unreal/unreal-getting-started.md [hunk 7]
```diff
@@ -2,74 +2,74 @@
...
 
 You’ll need the Magic Leap Hub to download the Magic Leap 2 Unreal SDK and examples. If you don’t already have the Magic Leap Hub installed:
 
-1. Go to the [developer tool download page](https://ml2-developer.magicleap.com/downloads) on the Magic Leap 2 developer portal and install the Windows version of the Magic Leap Hub.
+1. Go to the [developer tools download page](https://ml2-developer.magicleap.com/downloads) on the Magic Leap 2 developer portal and install the Windows version of the Magic Leap Hub.
 2. Run the installer and complete the setup wizard.
-3. Install the redistributables from the [Microsoft Visual C++ Redistributable latest supported downloads](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist) page.
-
-### Install the ML C SDK (optional)
+3. Go to the [Microsoft Visual C++ Redistributable latest supported downloads](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist) page. Install the X64 redistributables for Visual Studio 2015, 2017, 2019, and 2022.
```
Comment (aheaney) [inline docs/guides/unreal/unreal-getting-started.md:46]: @Babilinski Do you know does ML hub really not install the VC runtime libs for you?

Comment (Babilinski) [reply docs/guides/unreal/unreal-getting-started.md:46]: It does not :(

**Pull request merged**